code/modules/clothing/suits/ego_gear/non_abnormality/dawn.dm

```diff
@@ -0,0 +1,34 @@
+/obj/item/clothing/suit/armor/ego_gear/city/dawnleader
+	name = "dawn office leader jacket"
+	desc = "An armored jacket worn by the leader of dawn office."
+	icon_state = "dawnleader"
+	armor = list(RED_DAMAGE = 20, WHITE_DAMAGE = 50, BLACK_DAMAGE = 20, PALE_DAMAGE = 20)
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 80,
+							PRUDENCE_ATTRIBUTE = 80,
+							TEMPERANCE_ATTRIBUTE = 80,
+							JUSTICE_ATTRIBUTE = 80
+							)
+
+/obj/item/clothing/suit/armor/ego_gear/city/dawn
+	name = "dawn office jacket"
+	desc = "An armored jacket worn by dawn office fixers. This one is extremely well worn, and has been tailored many times."
+	icon_state = "dawn"
+	armor = list(RED_DAMAGE = 10, WHITE_DAMAGE = 40, BLACK_DAMAGE = 10, PALE_DAMAGE = 0)
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 60,
+							PRUDENCE_ATTRIBUTE = 60,
+							TEMPERANCE_ATTRIBUTE = 60,
+							JUSTICE_ATTRIBUTE = 60
+							)
+
+/obj/item/clothing/suit/armor/ego_gear/city/dawn/vet
+	name = "dawn office veteran attire"
+	desc = "An armored jacket and buisness dress worn by veteran dawn office fixers. This one is tailored for women, however a man might find the dress adventurous."
+	icon_state = "dawnvet"
+
+/obj/item/clothing/suit/armor/ego_gear/city/dawn/female
+	name = "dawn office dress"
+	desc = "An armored dress worn by dawn office fixers. The dress doesn't fit too tight, \
+		and leaves just enough for a full range of movement while leaving plenty to the imagination."
+	icon_state = "dawn_fem"
```

Comment:
Could probably incorporate these into adjustable ego gear at some point.

Reply:
This is only like this because Dawn office literally is just one guy and one girl...